src/components/Checkout/OrderTotals.vue

```diff
@@ -189,7 +185,6 @@ import { removeCredit } from '@/util/checkoutUtils';
 import showVerificationLightbox from '@/graphql/mutation/checkout/showVerificationLightbox.graphql';
 import KvTooltip from '@/components/Kv/KvTooltip';
 import VerifyRemovePromoCredit from '@/components/Checkout/VerifyRemovePromoCredit';
-import MatchedLoanKivaCredit from '@/components/Checkout/MatchedLoanKivaCredit';
```

Comment:
Looks like this component file can be deleted.

Reply:
Done